rules/falco_rules.yaml

```diff
@@ -453,7 +453,7 @@
     a shell configuration file has been modified (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline pcmdline=%proc.pcmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [file, mitre_persistence]
+  tags: [file, mitre_TA0003_persistence, mitre_TA0005_defense_evasion, mitre_T1562_impair_defenses, mitre_T1562.001_disable_or_modify_tools]
```

Comment:
Why does it fall into the `defense evasion` category?

Reply:
This is in connection to the next one in `Read Shell Configuration File`. By modifying these files defenses could be impaired (`T1562`) by disabling software security services.

Reply:
can you give me an example of modifying shell configuration leads to disabling security services?

Reply:
Solved